libs/ui-components/src/components/Catalog/CatalogPage.tsx

```
@@ -66,26 +66,24 @@ type CatalogEmptyStateProps = {
 const CatalogEmptyState = ({ hasFilters, showCatalogMgmt, isUpdating }: CatalogEmptyStateProps) => {
   const { t } = useTranslation();
 
-  const noResults = hasFilters || isUpdating;
+  if (isUpdating) {
+    return <ResourceListEmptyState icon={SearchIcon} titleText={t('Loading catalog items')}></ResourceListEmptyState>;
```

ESLint

[error] Empty components are self-closing

(react/self-closing-comp)